src/main/java/uk/gov/digital/ho/hocs/info/api/dto/WorkstackTypeDto.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.digital.ho.hocs.info.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.digital.ho.hocs.info.domain.model.WorkstackColumn;
+import uk.gov.digital.ho.hocs.info.domain.model.WorkstackType;
+
+import java.util.List;
+
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
+public class WorkstackTypeDto {
```

Comment:
I would just go with the name WorkstackDto, I'm not sure why we need the 'type' injected across this model. But bearing in mind this is across the entire platform now I think we can leave it as is now.

My understanding was that this object and configuration is going to be telling us how we display the given workstack. So 'Workstack' entity and 'WorkstackDto' could have been sufficient. It would of course have a 'type' property that would be it's unique identifier.